project-resources/README.md

```diff
@@ -0,0 +1,75 @@
+* [SECURITY_CONTACTS.md](templates/SECURITY_CONTACTS)
+  * a draft security contacts file to allow potential issue submitters to know
+      who they can expect to hear from or how to follow up on issues.
+* [ISSUE_TEMPLATE.md](ISSUE_TEMPLATE.md)
+  * a draft issue template to reminder issue submitters that potential
+      vulnerabilities **do not** get submitted as issues.
+* [incident-response.md](incident-response.md)
+  * a draft, detailed incident response plan that covers how to triage issues,
+      confirm vulnerabilities, leverage security advisories, and push a
+      patch/release.
+* [embargo-policy.md](embargo-policy.md)
+  * a draft embargo policy that outlines the time frame and conditions
+      surrounding disclosures.
+* [embargo.md](embargo.md)
+  * a draft embargo notification that details the contents a notification should
+    contain.
+
+Disclaimer: These resources are designed to be helpful to projects and
+organizations, they require customization and configuration by the project
+intending to use them. It does not prevent security issues from being found on a
+project, will not automatically resolve them, and does not place CNCF Security
+TAG as the responsible party. If changes are made to these templates, projects
+are not required to pull in a new update.
+
+## Dependabot Configuration
+
+All public repositories have dependabot graphs and alerts enabled. Projects are
+encouraged to also *[enable Dependabot security
+updates](https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/configuring-dependabot-security-updates)*
+and *create a [custom configuration for their
+project](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/enabling-and-disabling-version-updates)*.
+
+## CNCF project templates
+
+In addition to the security resources provided here, [CNCF's TAG Contributor
+Strategy](https://github.com/cncf/tag-contributor-strategy/blob/main/README.md)
+has put together an [initial project template
+collection](https://github.com/cncf/project-template) with information on
+getting started.
+
+
+## Updates to this directory
+
+The project-resources directory is intended to be a living directory to include
+a lot of resources and templates projects and the community finds useful for
+making projects more secure in a simple and easy-to-adopt fashion.  Updates,
+suggestions for updates, or discussions for updates should initiate with an
+[issue](https://github.com/cncf/tag-security/issues) and labeled with
+"suggestion".
+
+### Contributing updates
+
+All members of the community and projects are welcome to contribute updates to
+this directory. We ask potential contributors to refer to the existing content
+and discussions as guidance when determining the content of their updates.
+
+It is highly recommended that you seek peer review for your updates beyond that
```

Comment:
Trying to figure out what this means on top of the regular PR process? Is there a concern that wouldn't be covered with the current PR reviewing process?

Reply:
Not a concern, more an explicit statement as this may reach a broader audience who doesn't spend time going through our repo.